CleanCode.SolidPrinciples/InterfaceSegregation/ILobby.cs

```diff
@@ -9,24 +9,33 @@ namespace CleanCode.SolidPrinciples.InterfaceSegregation
     public interface ILobby
     {
         void AllIn();
+  
+        void AllOut();
+    }
+
+    public interface IPlayer
+    {
+        void PausePlayerTimer(string player);
```

Comment:
Chodziło mi, że Timer powinien się znaleść w osobnym interfejsie, bo teraz mieszasz odpowiedzialności. Kickowanie gracza i startowanie Timera to dwie różne sprawy